content/blog/production-ready-object-detection-model-training-workflow-with-hpe-machine-learning-development-environment.md

```diff
@@ -0,0 +1,1107 @@
+  --version 1.0 \
+  --model-file kserve_utils/torchserve_utils/model-xview.py \
+  --serialized-file kserve_utils/torchserve_utils/trained_model_stripped.pth \
+  --handler kserve_utils/torchserve_utils/fasterrcnn_handler.py \
+  --extra-files kserve_utils/torchserve_utils/index_to_name.json
+```
+
+After command finishes, run command to move file to our prepared `model-store/` directory:
+
+* `cp xview-fasterrcnn.mar kserve_utils/torchserve_utils/model-store -v`
+
+### Copy `config/` and `model-store/` folders to the K8S PVC Pod
+
+This is the directory structure needed to prepare our custom Pytorch Model for KServe inferencing:
+
+```
+├── config
+│   └── config.properties
+├── model-store
+│   ├── properties.json
+│   └── xview-fasterrcnn.mar
+```
+
+#### What the config.properties file looks like
+
+```
+inference_address=http://0.0.0.0:8085
+management_address=http://0.0.0.0:8085
+metrics_address=http://0.0.0.0:8082
+grpc_inference_port=7070
+grpc_management_port=7071
+enable_metrics_api=true
+metrics_format=prometheus
+number_of_netty_threads=4
+job_queue_size=10
+enable_envvars_config=true
+install_py_dep_per_model=true
+model_store=/mnt/models/model-store
+model_snapshot={"name": "startup.cfg","modelCount": 1,"models": {"xview-fasterrcnn": {"1.0": {"defaultVersion": true,"marName": "xview-fasterrcnn.mar","serialized-file":"trained_model_stripped.pth","extra-files":"index_to_name.json","handler":"fasterrcnn_handler.py","minWorkers": 1,"maxWorkers": 5,"batchSize": 1,"maxBatchDelay": 100,"responseTimeout": 120}}}}
+```
+
+#### What the properties.json looks like
+
+```
+[
+    {
+    "model-name": "xview-fasterrcnn",
+    "version": "1.0",
+    "model-file": "",
+    "serialized-file": "trained_model_stripped.pth",
+    "extra-files": "index_to_name.json",
+    "handler": "fasterrcnn_handler.py",
+    "min-workers" : 1,
+    "max-workers": 3,
+    "batch-size": 1,
+    "max-batch-delay": 100,
+    "response-timeout": 120,
+    "requirements": ""
+  }
+]
+```
+
+Note that we have a `config/` folder that includes a config.properties. This defines A. We also have a `model-store/` directory that contains are exported models and a `properties.json` file. We need this file for B
+
+Now we will run several kubectl commands to copy over these folders into our Pod and into the PVC defined directory
+
+* `kubectl cp kserve_utils/torchserve_utils/config/ model-store-pod:/pv/config/`
+* `kubectl cp kserve_utils/torchserve_utils/model-store/ model-store-pod:/pv/model-store/`
+
+Run these commands to verify the contents have been copied over to the pod.
+
+* `kubectl exec --tty model-store-pod -- ls /pv/config`
+* `kubectl exec --tty model-store-pod -- ls /pv/model-store`
+
+## Deploying model using a KServe InferenceService
+
+### Create Inference Service
+
+Below is the yaml definition that defines the KServe InferenceService that deploys models stored in the PVC. We already created a file that defines this PV in `k8s_files/torch-kserve-pvc.yaml`
```

Comment:
Instead of "We already created a file, change to "A file that defines this PV has already been created in 'k8s..."